flathub.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
you can remove this file because 

1. these are the only arches 
2. changing the delay isn't allowed.

Reply:
done